scss/tech-list.scss

```diff
@@ -1,8 +1,19 @@
 .tech-list {
-  display: flex;
+  align-items: center;
+  display: grid;
   flex-flow: row wrap;
+  grid-row-gap: 10px;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  justify-items: center;
   margin-bottom: 100px;
   max-width: 1050px;
   padding: 0 70px;
   width: 100%;
 }
+
+.tech-list div:nth-child(1) {
```

Comment:
- don't target `div` use classes instead, for this case `.tech`
- you can use `:first-child` instead of nth-child here